x-pack/filebeat/module/virustotal/livehunt/_meta/fields.yml

```diff
@@ -0,0 +1,924 @@
+      fields:
+        - name: flags
+          description: >
+            ELF Section List flags.
+          type: keyword
+        - name: name
+          description: >
+            ELF Section List name.
+          type: keyword
+        - name: physical_offset
+          description: >
+            ELF Section List offset.
+          type: keyword
+        - name: section_type
+          description: >
+            ELF Section List type.
+          type: keyword
+        - name: size
+          description: >
+            ELF Section List size.
+          format: bytes
+          type: long
+        - name: virtual_address
+          description: >
+            ELF Section List virtual address.
+          format: string
+          type: long
+    - name: exports
+      description: >
+        List of exported element names and types
+      release: beta
+      type: group
+      fields:
+        - name: name
+          description: >
+            Name of exported symbol
+          type: keyword
+          default_field: false
+        - name: type
+          description: >
+            Type of exported symbol
+          type: keyword
+          default_field: false
+    - name: imports
+      description: >
+        List of imported element names and types
+      release: beta
+      type: group
+      fields:
+        - name: name
+          description: >
+            Name of imported symbol
+          type: keyword
+          default_field: false
+        - name: type
+          description: >
+            Type of imported symbol
+          type: keyword
+          default_field: false
+    - name: shared_libraries
+      description: >
+        List of shared libraries used by this ELF object
+      type: keyword
+    - name: telfhash
+      description: >
+        telfhash hash for ELF files.
+      type: keyword
+    - name: flattened
+      default_field: false
+      description: >
+        Flattened ELF events from VirusTotal Intelligence Live Hunt results.
+      release: beta
+      type: group
+      fields:
+        - name: segment_list
+          description: >
+            ELF object segment list.
+          type: flattened
+- name: file.pe
+  type: group
+  fields:
+    - name: flattened
+      release: beta
+      type: group
+      fields:
+        - name: debug
+          default_field: false
+          description: >
+            Debug information, if present
+          type: flattened
+        - name: import_list
+          default_field: false
+          description: >
+            List of all imported functions
+          type: flattened
+        - name: sections
+          default_field: false
+          description: >
+            Data about sections of compiled binary PE
+          type: flattened
+        - name: resource_details
+          default_field: false
+          type: flattened
+          description: >
+            If the PE contains resources, some info about them
+        - name: resource_languages
```

Comment:
just wondering, does VT return language/type information tied to the specific resources it's enumerating? Because I would imagine this and the field below would show up in the `resource_details` albeit not aggregated.